packages/ListBox/src/components/Trigger/Trigger.styles.js

```diff
@@ -0,0 +1,140 @@
+import styled from "styled-components";
+import RawButton from "@paprika/raw-button";
+import RawButtonStyled from "@paprika/raw-button/lib/RawButton.styles";
+import tokens from "@paprika/tokens";
+
+const triggerStyles = `
+  align-items: center;
+  background-color: #ffffff;
+  border-radius: 3px;
+  border: 1px solid #d7d7d7;
+  box-sizing: border-box;
+  color: #3f3d3c;
+  display: block;
+  font-size: 14px;
+  padding: 8px; /* Define Trigger's height */
+  position: relative;
+  text-align: left;
+  transition: border-color 0.2s;
+  width: 100%;
+
+  /* RawButton span */
+  & > span{
+    width: calc(100% - 48px);
+  }
+`;
+
+const triggerStylesProps = () => props => {
+  const isDisabled = props.isDisabled ? `color: ${tokens.color.blackLighten60};` : "";
+  return props.isInline
+    ? `
+      ${triggerStyles}
+      border-bottom-left-radius: 0;
+      border-bottom-right-radius: 0;
+      ${isDisabled}
+    `
+    : `
+      & > [data-raw-button="trigger"] {
+        ${triggerStyles}
+        ${isDisabled}
+      }
+    `;
+};
+
+export const ListBoxTriggerStyled = styled.div`
+  position: relative;
+  ${triggerStylesProps()}
+`;
+
+export const TriggerActionIconsContainer = styled.div`
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  position: absolute;
+  right: 0;
+  top: 0;
+  height: 100%;
+  pointer-events: none;
+`;
+
+export const RawButtonClearButtonStyled = styled(RawButton)`
+  ${RawButtonStyled}
+  && {
```

Comment:
What does the double ampersand do??

Reply:
add specificity to the style so no need to use important https://github.com/styled-components/styled-components-website/issues/187#issuecomment-352727285